modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/RoleManager.java

```diff
@@ -48,7 +48,8 @@ public interface RoleManager {
      * @throws NotImplementedException NotImplementedException.
      * @throws BadRequestException     BadRequestException.
      */
-    Role createRole(Role role) throws CharonException, ConflictException, NotImplementedException, BadRequestException;
+    Role createRole(Role role) throws CharonException, ConflictException, NotImplementedException,
+            BadRequestException, ForbiddenException;
```

Comment:
won't this introduce any behavioural change? Then we have to do a version bump

Reply:
So far ForbiddenException was not thrown. 

Now also throwing for the cases introduced in this effort.

Do we still need to consider this as a behavioural change?